unlock-app/src/pages/locks/metadata.tsx

```diff
@@ -0,0 +1,22 @@
+import { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import { AppLayout } from '~/components/interface/layouts/AppLayout'
+import { UpdateMetadataForm } from '../../components/interface/locks/metadata'
+
+const Metadata: NextPage = () => {
+  const router = useRouter()
+  const lockAddress = router.query.lockAddress?.toString()?.toLowerCase()
+  const network = Number(router.query.network)
+  const keyId = router.query.keyId?.toString()?.toLowerCase()
+  return (
+    <AppLayout>
+      <UpdateMetadataForm
```

Comment:
Can you please handle the case when `lockAddress` (or `network`) is not set by using the lock picker that @kalidiagne created?